ml_metadata/tools/mlmd_bench/fill_types_workload_test.cc

```diff
@@ -29,52 +31,274 @@ namespace ml_metadata {
 namespace {
 
 constexpr int64 kNumOperations = 100;
+constexpr int64 kNumOperationsForMakeUpCases = 200;
+
+// Generates FillTypes Workload where `update` indicates insert or update cases.
+void GenerateFillTypesWorkload(
```

Comment:
to address the test coverage, instead of populating these vectors, you can consider using TEST_P with parameters. 

in our case, there're two parameters
1. bool update
2. specification

Please try to reuse previous version of the test, and improve it with TEST_P, 
learning the way to write TEST_P helps testing future work items too (as specification is a common pattern in the design). 

See [1] and [2] for documents and examples

[1] https://github.com/google/googletest/blob/70b90929b1da20580cad9ed996397cf04ef8f16d/googletest/docs/advanced.md#value-parameterized-tests
[2] https://github.com/google/googletest/search?l=C%2B%2B&q=TEST_P

Reply:
Yes, instead of set the `update` and `specification` as the parameters, I believe it will be easier if I use the workload config. as the parameter directly since we are going to initialize the config. any way using `update` and `specification`.

Also, I separate the insert and update tests due to their very different tests method. wdyt?